Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: env_setup
+
+env_setup: ## Setup development environment
```

Comment:
How about simply `setup`?

Reply:
I prefer to keep it as `env_setup` as the `setup` is bigger than the `env`, maybe later we have another `setup step` so we could separate it to different `step`, and if we want to combine that into 1 setup step, Makefile allow us to do that

```
setup: env_setup another_setup

env_setup:
 - xxx

another_setup:
 - yyy
```

Reply:
To me, it feels like inheriting from the Rails project. And actually there is no environment setup at all, just the db 😆 How about `docker_setup` then? or `db_setup`?

Reply:
We can go with `docker_setup` @olivierobert